config/features.yml

```diff
@@ -7,6 +7,10 @@
 #  or `cookie_id` of you wish to use the Google Analytics id as the unique identifier.
 
 features:
+  dashboard_show_covid19_alert:
```

Comment:
will be used to gate this feature https://github.com/department-of-veterans-affairs/vets-website/pull/11952